apps/dotcms-ui/src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.ts

```diff
@@ -48,15 +48,17 @@ export class DotThemeSelectorDropdownComponent
     ) {}
 
     ngOnInit(): void {
-        this.currentSiteIdentifier = this.siteService.currentSite.identifier;
+        setTimeout(() => {
```

Comment:
this is needed, otherwise when you create a template via url hook (enter the url on browser) `http://localhost:8080/dotAdmin/#/templates/new/designer` an error will be thrown since the siteService is not loaded in the main app yet